appveyor.yml

```diff
@@ -8,6 +8,8 @@ environment:
   PASSWORD:
     secure: mqbXr4+cn3l9YNyKhOezbA==
   matrix:
+    - PRODUCT: tests
```

Comment:
I prefer not to move this at the moment, I'm usually integrated in the android build...